tests/publiccloud/run_ltp.pm

```diff
@@ -141,18 +141,20 @@ sub run {
 
     assert_script_run($log_start_cmd);
 
-    my $cmd = 'perl -I runltp-ng runltp-ng/runltp-ng ';
-    $cmd .= '--logname=ltp_log --verbose ';
-    $cmd .= '--timeout=1200 ';
-    $cmd .= '--run ' . get_required_var('LTP_COMMAND_FILE') . ' ';
-    $cmd .= '--exclude \'' . get_var('LTP_COMMAND_EXCLUDE') . '\' ' if get_var('LTP_COMMAND_EXCLUDE');
-    $cmd .= '--backend=ssh';
+    zypper_call("in -y python3-paramiko python3-scp");
+ 
+    my $cmd = 'python3 runltp-ng/runltp-ng ';
+    $cmd .= "--json-report=$root_dir/result.json ";
+    $cmd .= '--verbose ';
+    $cmd .= '--suite-timeout=2400 ';
+    $cmd .= '--run-suite ' . get_required_var('LTP_COMMAND_FILE') . ' ';
+    $cmd .= '--skip-tests \'' . get_var('LTP_COMMAND_EXCLUDE') . '\' ' if get_var('LTP_COMMAND_EXCLUDE');
+    $cmd .= '--sut=ssh';
     $cmd .= ':user=' . $instance->username;
+    $cmd .= ':sudo=1';
     $cmd .= ':key_file=' . $instance->ssh_key;
     $cmd .= ':host=' . $instance->public_ip;
     $cmd .= ':reset_command=\'' . $reset_cmd . '\'';
-    $cmd .= ':ssh_opts=\'-o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no\' ';
```

Comment:
why we not passing this ssh options anymore ? I think ssh tunnel without them may get into trouble when host will change host signature ( and this happens a lot in public cloud )

Reply:
`ssh` command is no longer used, so SSH options are not available
https://github.com/acerv/runltp-ng/blob/945196ca4b6814ce7efdb04afcf9c0be022d4ea5/ltp/ssh.py#L87

But you can open an issue request and provide some new SSH supports to add. We can add more specific options to the ssh SUT option.

Reply:
we do need to pass this "-o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no" and it is NO GO for us before it won't be implemented ( failure rate will be too high ... )

Reply:
"StrictHostKeyChecking=no" can be implemented here https://github.com/acerv/runltp-ng/blob/945196ca4b6814ce7efdb04afcf9c0be022d4ea5/ltp/ssh.py#L89

Using:

`self._client.set_missing_host_key_policy(MissingHostKeyPolicy())`

This can be implemented adding an option to set host key policy.